courses/machine_learning/deepdive/08_image/mnist_linear.ipynb

```diff
@@ -9,9 +9,7 @@
    "source": [
```

Comment:
<p>In create_input_fn do we really need to return a function that returns a dataset? Doesn't Keras model.fit also work if we pass the dataset instance directly (as opposed to a function that returns the instance)? If so, we should do that instead, since it's simpler - and rename the function accordingly then.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/551/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
Made it that was so it was reusable with estimator. Since that's gone now, this got simplified.

Reply:
<p>I'd give a more concrete name to the input_fn now that this notions has kind of disappeared, like load_dataset or create_dataset. </p>

Reply:
<p>Whoops, meant to do that. Good catch!</p>